experimental/traffic-portal/src/app/core/parameters/table/parameters-table.component.spec.ts

```diff
@@ -165,4 +165,29 @@ describe("ParametersTableComponent", () => {
 		expect(item.fragment).toBeUndefined();
 		expect(item.newTab).toBeTrue();
 	});
+
+	it("generates 'View Profiles' context menu item href", () => {
+		const item = component.contextMenuItems.find(i => i.name === "View Profiles");
+		if (!item) {
+			return fail("missing 'View Profiles' context menu item");
+		}
+		if (isAction(item)) {
+			return fail("expected a link, not an action");
+		}
+		if (!item.href) {
+			return fail("missing 'href' property");
+		}
+		if (typeof(item.href) !== "string") {
+			return fail("'View Profiles' context menu item should use a static string to determine href, instead uses a function");
+		}
+		expect(item.href).toBe("/core/profiles");
+		if (typeof(item.queryParams) !== "function") {
+			return fail(
+				`'View Profiles' context menu item should use a function to determine query params, instead uses: ${item.queryParams}`
+			);
+		}
+		expect(item.queryParams(testParameter)).toEqual({hasParameter: testParameter.id});
+		expect(item.fragment).toBeUndefined();
+		expect(item.newTab).toBeTrue();
```

Comment:
Should this be opening in a new tab? I don't believe other, similar links do that.